PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/model/dto/ImageModelDto.java

```diff
@@ -0,0 +1,30 @@
+package pl.polsl.photoplus.model.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+import pl.polsl.photoplus.annotations.Patchable;
+import pl.polsl.photoplus.annotations.validators.Unique;
+import pl.polsl.photoplus.services.controllers.ImageService;
+
+import javax.persistence.Lob;
+import javax.validation.constraints.NotBlank;
+
+@Getter
+@Setter
+public class ImageModelDto extends AbstractModelDto<ImageModelDto> {
```

Comment:
Is this needed in picture controller we do not receive this pojo?

Reply:
it's needed to use generic service